interactive_engine/src/executor/jna/src/executor/gaia/gaia_server.rs

```diff
@@ -0,0 +1,127 @@
+use maxgraph_store::db::api::{GraphConfig, GraphResult, GraphError};
+use std::sync::Arc;
+use maxgraph_store::db::graph::store::GraphStore;
+use std::net::SocketAddr;
+use maxgraph_runtime::store::v2::global_graph::GlobalGraph;
+use gaia_pegasus::Configuration;
+use maxgraph_store::db::api::GraphErrorCode::EngineError;
+use tokio::runtime::Runtime;
+use tokio::sync::oneshot;
+use pegasus_server::service::Service;
+use pegasus_server::rpc::start_rpc_server;
+use pegasus_network::manager::SimpleServerDetector;
+use pegasus_network::config::NetworkConfig;
+use crate::executor::gaia::initialize_job_compiler;
+use std::collections::HashMap;
+use maxgraph_store::api::PartitionId;
+
+pub struct GaiaServer {
+    config: Arc<GraphConfig>,
+    graph: Arc<GlobalGraph>,
+    detector: Arc<SimpleServerDetector>,
+}
+
+impl GaiaServer {
+    pub fn new(config: Arc<GraphConfig>) -> GraphResult<Self> {
```

Comment:
At least expect() instead of `unwrap()`. In addition, I am wondering, if the function returns a result, why you can not throw the error of `get_storage_option` and `parse`